core/templates/components/oppia-angular-root.component.ts

```diff
@@ -463,15 +463,18 @@ import { WrittenTranslationObjectFactory } from
   'domain/exploration/WrittenTranslationObjectFactory';
 import { WrittenTranslationsObjectFactory } from
   'domain/exploration/WrittenTranslationsObjectFactory';
+import { AppConstants } from 'app.constants';
 
 @Component({
   selector: 'oppia-angular-root',
-  template: ''
+  templateUrl: './oppia-angular-root.component.html'
 })
 export class OppiaAngularRootComponent implements AfterViewInit {
   @Output()
     public initialized: EventEmitter<void> = new EventEmitter();
-  static ajsTranslate;
+  direction: string = 'ltr';
+  currentLang = 'en';
```

Comment:
I have found that we dynamically interpolate the `lang` attribute of HTML tag but do not update it when site language changes.
@oppia/core-maintainers Is this the desired behavior? If yes, then why we have used dynamic interpolation for the lang attribute.

Reply:
Considering above behaviour as unexpected, I have fix this in this PR.